### PR TITLE
add an example how to get swiss altitude

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: documentation/how-to/gnss.en.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ For example:
 For the transformation from ETRS89 (reference ellipsoid GPS) to NAP (Dutch) users can download
 the file [nlgeo2018.gtx from NSGI](https://www.nsgi.nl/rdnaptrans) and put it in the directory.
 
+To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGC from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip). Following the download, users are advised to perform a conversion of the file labeled "chgeo04_htrans_lv95.arg" to "chgeo04_htrans_lv95.gtx".
```
Comment (suricactus) [inline documentation/how-to/gnss.en.md:88]:
```suggestion
To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGC from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip). Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.arg` to `chgeo04_htrans_lv95.gtx`.
```

File: documentation/how-to/gnss.en.md [hunk 2]
```diff
@@ -85,6 +85,8 @@ For example:
 For the transformation from ETRS89 (reference ellipsoid GPS) to NAP (Dutch) users can download
 the file [nlgeo2018.gtx from NSGI](https://www.nsgi.nl/rdnaptrans) and put it in the directory.
 
+To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGC from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip). Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.arg` to `chgeo04_htrans_lv95.gtx`.
```
Comment (suricactus) [inline documentation/how-to/gnss.en.md:88]:
```suggestion
To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGC from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip).
Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.arg` to `chgeo04_htrans_lv95.gtx`.
```

File: documentation/how-to/gnss.en.md [hunk 3]
```diff
@@ -85,6 +85,9 @@ For example:
 For the transformation from ETRS89 (reference ellipsoid GPS) to NAP (Dutch) users can download
 the file [nlgeo2018.gtx from NSGI](https://www.nsgi.nl/rdnaptrans) and put it in the directory.
 
+To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGC from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip).
```
Comment (m-kuhn) [inline documentation/how-to/gnss.en.md:88]:
```suggestion
To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGD from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip).
```

Follow file naming

File: documentation/how-to/gnss.en.md [hunk 4]
```diff
@@ -85,6 +85,9 @@ For example:
...
+Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.arg` to `chgeo04_htrans_lv95.gtx`.
```
Comment (m-kuhn) [inline documentation/how-to/gnss.en.md:89]:
```suggestion
Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.agr` to `chgeo04_htrans_lv95.gtx`.
```

The file in the zip is called `.agr`

File: documentation/how-to/gnss.en.md [hunk 5]
```diff
@@ -85,6 +85,9 @@ For example:
 For the transformation from ETRS89 (reference ellipsoid GPS) to NAP (Dutch) users can download
 the file [nlgeo2018.gtx from NSGI](https://www.nsgi.nl/rdnaptrans) and put it in the directory.
 
+To obtain precise altitude data for Cadastral Surveying in Switzerland, users can access the file correction of the vertical grid shift through [Geoid OGD from Swisstopo](https://cms.geo.admin.ch/ogd/geodesy/Geoid_OGD.zip).
+Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.agr` to `chgeo04_htrans_lv95.gtx`.
```
Comment (m-kuhn) [inline documentation/how-to/gnss.en.md:89]:
```suggestion
Following the download, users are advised to perform a conversion of the file labeled `chgeo04_htrans_lv95.agr` to `chgeo04_htrans_lv95.gtx`. The QGIS processing algorithm `gdal:translate` (convert format) can be used for that.
```

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**